packages/babel-plugin-component-annotate/src/index.ts

```diff
@@ -61,6 +61,12 @@ type AnnotationPlugin = PluginObj<AnnotationPluginPass>;
 export default function componentNameAnnotatePlugin({ types: t }: typeof Babel): AnnotationPlugin {
   return {
     visitor: {
+      Program: {
+        enter(path, state) {
+          const fragmentContext = collectFragmentContext(path);
+          state['sentryFragmentContext'] = fragmentContext;
```

Comment:
We should update `AnnotationPluginPass` to strongly type `sentryFragmentContext`.